computation_scripts/master.py

```diff
@@ -327,43 +351,55 @@ def check_installations() -> None:
         exit()
 
 
-def main(retro_zarr: str, nc: str, ) -> None:
-    """
-    Assumes docker is installed and this command was run: docker pull chdavid/rapid.
-    Assumes AWS CLI and s5cmd are likewise installed. 
-
-    Executes the main workflow for running the RAPID model.
-
-    Args:
-        retro_zarr (str): The path to the retro zarr dataset.
-        nc (str): The path to the nc file.
+def concatenate_outputs() -> None:
+    # Build the week dataset
+    qouts = natsort.natsorted(glob.glob(os.path.join(HOME, 'data', 'outputs', '*', 'Qout*.nc')))
+
+    with xr.open_zarr(local_zarr) as retro_ds:
+        chunks = retro_ds.chunks
+
+    with xr.open_mfdataset(
+            qouts,
+            combine='nested',
+            concat_dim='rivid',
+            preprocess=drop_coords
+    ) as new_ds:
```

Comment:
I've had issues when creating the dataset suitable to append to the zarr. Make sure that we don't need this line: ".reindex(rivid=xr.open_zarr(local_zarr)" attached here